MANIFEST.in

```diff
@@ -1,5 +1,8 @@
-include *.rst LICENSE Makefile tox.ini .coveragerc
+include *.rst LICENSE Makefile tox.ini .coveragerc app.wsgi
 include kinto/config/kinto.tpl
-recursive-include kinto *.sql *.html
-recursive-include docs *.rst *.png *.svg
+include dev-requirements.txt
+recursive-include kinto *.sql *.html *.ini
+recursive-include docs *.rst *.png *.svg *.css *.html conf.py piwik.js
 prune docs/_build/*
+prune loadtests
+prune talks
```

Comment:
I'm not clear on the difference between `prune` and `recursive-exclude`. The [documentation](https://docs.python.org/2/distutils/sourcedist.html#manifest) doesn't suggest that there should be any difference between the two commands. The only difference I can see is that `recursive-exclude` requires that we specify every different kind of file -- for `loadtests`, this would be `.ini` files, `.py` files, `.json`, and a Makefile. For `talks`, it would be `.rst`, `.css`, a couple stray `.cfg`s, some `.png`s, `.jpg`s, and `.svg`s, and another Makefile. It seems to me like `prune` suggests that no matter what file shows up in `talks`, we don't want it in the package, but `recursive-exclude` suggests that we only want to exclude certain kinds of files and other kinds of files in `talks` would be OK. But I don't know anything at all about this so I'll do whatever you recommend :)